server.go

```diff
@@ -163,21 +163,41 @@ func (s *server) UpdatePin(ctx context.Context, request v0.UpdatePinRequestObjec
 		user = usersResponse.Payload[0]
 	}
 
+	if len(s.cfg.Nuki.SmartLockDevices) == 0 && s.cfg.Nuki.SmartLockDevice == 0 {
+		return nil, fmt.Errorf("no smart lock device configured")
+	}
+	// Retro compatibility
+	if len(s.cfg.Nuki.SmartLockDevices) == 0 {
+		s.cfg.Nuki.SmartLockDevices = append(s.cfg.Nuki.SmartLockDevices, s.cfg.Nuki.SmartLockDevice)
```

Comment:
I think we would make this list bigger and bigger with each incoming request.

We should not temper with the config in the handler. Only at program start